fixieai/cli/agent/commands.py

```diff
@@ -447,7 +455,17 @@ def _server_process(
             "--port",
             str(port),
             "--factory",
-        ],
+        ]
+    elif agent_type == fixieai.client.FixieEnvironment.NODEJS:
+        args = [
+            agent_exe,
+            "ts-node",
```

Comment:
I saw this on build PR as well -- why do we need to use `ts-node`? Don't have a particular problem with it, just seems odd since we're ostensibly executing a `.js` file.

Reply:
I just copied this from the previous SDK branch. Should I just run "node" here?